yt/yt/tests/integration/controller/test_merge_operation.py

```diff
@@ -2272,6 +2272,40 @@ def test_mr_any_value_with_hunks(self, optimize_for):
 
         assert read_table("//tmp/t") == rows
 
+    @authors("achulkov2")
+    def test_chunk_slice_statistics(self):
```

Comment:
Let's test that we still throw the same error when trying to provide key bounds on non-sorted tables.